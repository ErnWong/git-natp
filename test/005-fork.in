
            I
      .-E--<
     /   K  J
  ,-B--F'-L
A<--C--G
  `-D   *.
     \    M master
      H
       `.  ,N
         `<
           `O
