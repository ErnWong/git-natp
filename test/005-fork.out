A
B A
C A
D A
E B
F B
G C
H D
I E
J E
K F
L F
M G
N H
O H
